#321 Turbo Tool for Houdini (#325)
Inbibo/adonisfx_docs: docs/houdini/tools/turbo_tool.md

</figure>

> [!NOTE]
> - Fascia and fat meshes must have the same topology for the AdnFat deformer to be created by AdnTurbo.
> - AdnTurbo can also be executed via Python scripting. For more details, please refer to the [Turbo Script page](../scripts/turbo).

## Result

As a result of executing the tool by providing the geometries for all the layers, the following nodes will be created:

- An AdnMuscle for each muscle geometry with the mummy geometry as target.
- An AdonisFX locator and sensor for each AdnMuscle to drive the muscle activation.
- An AdnGlue node with all the muscles merged as input.
- An AdnSkin node for the fascia geometry with the mummy and glue as targets.
- An AdnRelax node applied on top of the fascia AdnSkin.
- An AdnFat node for the fat geometry with the fascia geometry as base mesh.
- An AdnRelax node applied on top of the AdnFat.
- An AdnSkin node for the skin geometry.

## Limitations

- The **Glue Layer** cannot be bypassed. This means that for the **Fascia Layer** to be enabled, the **Glue Layer** checkbox must be checked.
- The default values that AdnTurbo will use to configure each deformer cannot be customized.

> - Currently, the primitive attribute name used to separate the muscles must be `path` and it cannot be change when running AdnTurbo through the UI.

Reviewer: ```suggestion
- The primitive attribute name used to separate the muscles defaults to `path` and cannot be changed when running AdnTurbo through the UI.
```

Author: Done.
